academy/3-my-own-chain/migration.md

```diff
@@ -513,7 +515,7 @@ ErrInvalidDateAdded = sdkerrors.Register(ModuleName, 1120, "dateAdded cannot be
 
 </ExpansionPanel>
 
-<ExpansionPanel title="Add functions to (de)serialize WinningPlayerParsed">
+<ExpansionPanel title="Add functions to (de-serialize WinningPlayerParsed">
```

Comment:
```suggestion
<ExpansionPanel title="Add functions to (de)serialize WinningPlayerParsed">
```

Writing _(de)action-word_ is a common way of saying that we are going to talk about both _serialize_ and _deserialize_.